tutorials/kotlin.md

```diff
@@ -64,7 +64,36 @@ repositories {
 }
 ```
 
-Please note that you should **not** include `kotlin-runtime` and `kotlin-stdlib` jars with your plugin because Kotlin guarantees backward- and forward- binary compatibility. 
+Please note that you should **not** include `kotlin-runtime` and `kotlin-stdlib` jars with your plugin because Kotlin guarantees backward- and forward- binary compatibility.
+
+### 3.1. Use Kotlin to write gradle script
+
+After gradle 4.4, gradle starts supporting `build.gradle.kts`, an alternative to `build.gradle` written in Kotlin.
+
+It's not yet very well-documented but there're many good resources for learning how to write build scripts for an IntelliJ plugin with Kotlin script, like [intellij-rust](https://github.com/intellij-rust/intellij-rust/blob/master/build.gradle.kts), [julia-intellij](https://github.com/ice1000/julia-intellij/blob/master/build.gradle.kts), [covscript-intellij](https://github.com/covscript/covscript-intellij/blob/master/build.gradle.kts) or [zig-intellij](https://github.com/ice1000/zig-intellij/blob/master/build.gradle.kts).
```

Comment:
I would not make any statements about some technology not being well-documented (it's always subjective, and even if true, will likely change soon).

Reply:
Get